app/move/index.js

```diff
@@ -40,13 +56,30 @@ router.use(
   wizard(createSteps, createFields, createConfig)
 )
 router.get('/:moveId', protectRoute('move:view'), view)
+
 router.get('/:moveId/confirmation', protectRoute('move:create'), confirmation)
 router.use(
   '/:moveId/cancel',
   protectRoute('move:cancel'),
   wizard(cancelSteps, cancelFields, cancelConfig)
 )
 
+if (FEATURE_FLAGS.EDITABILITY) {
```

Comment:
If its not too much trouble it might be worth seeing if you can move the addition of this feature flag to the other commit. That way when we remove the flag we can reverse that commit and remove any code needed for the feature flag.

Reply:
I presume you mean this commit (https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/450/commits/57065c6c64976a2cbafd5a000c44d2f90a59473c). Could you explain what you mean by reverse the commit? Rebase so it was never there or...?

Reply:
Yep, that commit. And sorry, I meant [`revert`](https://git-scm.com/docs/git-revert), not reverse. Ie, when we're done with the feature flag because the feature is live everywhere, it makes it easy to remove the associated code.

Reply:
OK. But I'll do that last when all the other changes have been made and squashed.

Reply:
Ack - the e2e tests have the flag in too. I'll reverse the order of those commits when the time comes...